snowplowtracker-scm-1.rockspec

```diff
@@ -18,8 +18,9 @@ dependencies = {
   "lua >= 5.1",
   "lua-curl >= 0.3.13-1",
   "base64 >= 1.5-2",
-  "lua_uuid >= 0.2.0-2",
-  "lunajson >= 1.2.3-1"
+  "uuid >= 0.2.0-2",
```

Comment:
Good call to replace `lua_uuid` with `uuid`, it seems much more popular and maintained! However, shouldn't we change the min requirement for the latest version 0.3-1?